fw/beef.c

```diff
@@ -315,6 +362,10 @@ void update_lighting(uint16_t led_state) {
   }
 }
 
-bool is_pressed(uint8_t button_bit) {
-  return button_state & (1 << button_bit);
+bool is_pressed(uint16_t button_bits) {
+  return button_state & button_bits;
+}
+
+bool is_pressed_strict(uint16_t button_bits) {
+  return button_state == button_bits;
```

Comment:
I don't see any remaining usage of `is_pressed`. Please remove `is_pressed` and rename `is_pressed_strict` -> `is_pressed`

Reply:
I figured that `is_pressed()` could be useful later on, so I think it's fine if we kept this function.

Reply:
Sure